frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -362,9 +362,10 @@ const ManagePolicyPage = ({
   );
 
   const refetchPolicies = (teamId?: number) => {
-    refetchGlobalPolicies();
     if (teamId) {
       refetchTeamPolicies();
+    } else {
+      refetchGlobalPolicies(); // Only call on global policies as this is expensive
```

Comment:
This makes sense in light of recent performance issues. I don't recall offhand, but were there any places where this function was being used to refresh the policies after certain user actions? Also, we might want to keep an eye out for cases where the global policies results are getting stale. One side effect of the old approach was that the policies were updated every time the team dropdown changed. I think it's ok to merge as-is, but wanted to highlight these as things to watch for in QA.